bot/exts/help_channels/_cog.py

```diff
@@ -78,7 +78,10 @@ def __init__(self, bot: Bot):
         self.channel_queue: asyncio.Queue[discord.TextChannel] = None
         self.name_queue: t.Deque[str] = None
 
-        self.last_notification: t.Optional[arrow.Arrow] = None
+        # Notifications
+        self.notify_interval_seconds = (constants.HelpChannels.notify_minutes * 60)
+        self.last_none_remaining_notification = arrow.get('1815-12-10T18:00:00.00000+00:00')
+        self.last_running_low_notification = arrow.get('1815-12-10T18:00:00.00000+00:00')
```

Comment:
Are these just arbitrarily low dates? If so doing `arrow.get(datetime.min)` might make it more clear what this is trying to do.

Reply:
it is Ada Lovelace's birthday as a little "easter egg". it is just a very old date, so we don't have to use `Optional` typing.

Would you like for me to change it or comment it?

Reply:
Yea, I think just a comment that it's just an arbitrarily old date is fine.